salesforce-gui/src/main/java/org/fundacionjala/salesforce/unitTest/utils/Utils.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.salesforce.unitTest.utils;
+
+import org.fundacionjala.salesforce.constants.Constants;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Base64;
+
+/**
+ * [SL] This class contains utilities for the project.
+ */
+public final class Utils {
```

Comment:
This name could be changed as decodingUtils, the name "Utils" is not descriptive.